docs/concepts/api-extension/custom-resources.md

```diff
@@ -72,6 +223,318 @@ implementations for your custom resources by writing and deploying your own stan
 The main API server delegates requests to you for the custom resources that you handle,
 making them available to all of its clients.
 
+### Choosing which method to use to add a User-defined Resource
+
+CRDs is easier to use.  Aggregated APIs are more flexible.  Choose the one that best meets your needs.
+
+Typically, CRDs are a good fit if:
+
+
+
+*   You are have a handful of fields 
+*   You are using the resource within your company, or as part of a small open-source project (as opposed to a commercial product)
+
+<table>
+  <tr>
+   <td colspan="2" >
+<strong>Ease of use comparison</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>CRDs
+   </td>
+   <td>Aggregated API
+   </td>
+  </tr>
+  <tr>
+   <td>Do not require programming.
+<p>
+For writing a controller for the custom resources, user can chose any language for the controller.
+   </td>
+   <td>Requires programming in Go and building binary and image.
```

Comment:
To clarify the distinction from the next sentence, add "... for the new API server".